packages/cli-platform-apple/src/commands/runCommand/getBuildPath.ts

```diff
@@ -1,38 +1,15 @@
-import child_process from 'child_process';
-import {IOSProjectInfo} from '@react-native-community/cli-types';
-import {CLIError, logger} from '@react-native-community/cli-tools';
-import chalk from 'chalk';
+import {CLIError} from '@react-native-community/cli-tools';
+import path from 'path';
+import {ApplePlatform} from '../../types';
 
 export async function getBuildPath(
-  xcodeProject: IOSProjectInfo,
-  mode: string,
-  buildOutput: string,
-  scheme: string,
-  target: string | undefined,
+  buildSettings: any,
```

Comment:
This should be properly typed. We should avoid using `any` in new code.

Reply:
Yeah, I created second [PR](https://github.com/react-native-community/cli/pull/2234), where it is properly handled - I'll rebase after merging 👍